.eslintrc

```diff
@@ -49,7 +49,15 @@
       { "selector": "variable", "modifiers": ["destructured"], "format": null },
       { "selector": "variable", "types": ["boolean"], "format": ["PascalCase"], "prefix": ["is", "should", "has", "can", "did", "will", "does"] },
       { "selector": "variableLike", "format": ["camelCase"] },
-      { "selector": ["function", "variable"], "format": ["camelCase"] }
+      { "selector": ["function", "variable"], "format": ["camelCase"] },
+      {
+        "selector": "variable",
+        "format": ["PascalCase"],
+        "filter": {
+          "regex": "^(Erc20Permit|Erc721Permit)$",
```

Comment:
Thank you for your pull but this should be generalized for every identifier not just those two.